src/main/scala/cromwell/engine/package.scala

```diff
@@ -61,6 +64,19 @@ package object engine {
     }
   }
 
+  implicit class EnhancedExecution(val execution: Execution) extends AnyVal {
```

Comment:
Specifically the extension of AnyVal makes it a value class which means that the underlying JVM byte code is operating directly on the execution and not the wrapper class